index.js

```diff
@@ -32,7 +33,7 @@ CacheBreaker.prototype.cacheBreakPath = function(base, resource) {
   var extname = path.extname(resource);
   var basename = path.basename(resource, extname);
 
-  return path.join(dirname, basename + '.' + cs.substring(0, 10) + extname);
+  return url.resolve(dirname + '/', basename + '.' + cs.substring(0, 10) + extname);
```

Comment:
I believe it is because `path` is designed to be used for file paths. From the documentation:
```This module contains utilities for handling and transforming file paths. ```

Whereas `url` is meant to be used for URL handling:
```This module has utilities for URL resolution and parsing.```